src/store/polls/index.ts

```diff
@@ -16,6 +16,7 @@ export const state: PollsStateInterface = {
     polledHouses: [],
     status: 'active',
     type: '',
+    acceptanceCriteria: '',
```

Comment:
Is there an option, when acceptanceCriteria can be empty?

Reply:
acceptanceCriteria can be empty when user doesn't select poll